functions.php

```diff
@@ -5,83 +5,60 @@
  * @package Go
  */
 
-/**
- * Theme constants.
- */
-define( 'GO_VERSION', '1.8.1' );
+// Load deactivation modal.
+define( 'GO_VERSION', '2.0.0' );
 define( 'GO_PLUGIN_DIR', get_template_directory( __FILE__ ) );
 define( 'GO_PLUGIN_URL', get_template_directory_uri( __FILE__ ) );
+require_once get_parent_theme_file_path( 'includes/class-go-theme-deactivation.php' );
 
 /**
- * AMPP setup, hooks, and filters.
+ * Sets up theme defaults and registers support for various WordPress features.
  */
-require_once get_parent_theme_file_path( 'includes/amp.php' );
+function go_theme_setup() {
+	add_theme_support( 'wp-block-styles' );
+	add_editor_style( 'style.css' );
 
-/**
- * Core setup, hooks, and filters.
- */
-require_once get_parent_theme_file_path( 'includes/core.php' );
+	// Disable CoBlocks Site Design controls.
+	delete_option( 'coblocks_site_design_controls_enabled' );
+}
 
-/**
- * Customizer additions.
- */
-require_once get_parent_theme_file_path( 'includes/customizer.php' );
+add_action( 'after_theme_setup', 'go_theme_setup' );
 
 /**
- * Custom template tags for the theme.
+ * Enqueue theme styles.
  */
-require_once get_parent_theme_file_path( 'includes/template-tags.php' );
+function go_theme_styles() {
+	wp_enqueue_style(
+		'style',
+		get_stylesheet_uri(),
+		array(),
+		GO_VERSION
+	);
+}
+add_action( 'wp_enqueue_scripts', 'go_theme_styles' );
 
-/**
- * Pluggable functions.
- */
-require_once get_parent_theme_file_path( 'includes/pluggable.php' );
+// Migration process.
+if ( isset( $_GET['migrate'] ) ) { // phpcs:ignore WordPress.Security.NonceVerification
+	require_once get_parent_theme_file_path( '/includes/class-classic-conversion.php' );
```

Comment:
Just for my understanding, Is the plan for this to be fully automated once released? Or maybe a button to do the migration?

Reply:
Yes, this condition is temporary and for testing.